src/System.Management.Automation/engine/parser/tokenizer.cs

```diff
@@ -1229,23 +1229,125 @@ private int SkipBlockComment(int i)
             return i;
         }
 
-        private static char Backtick(char c)
+        private string Backtick(char c)
```

Comment:
One potential issue with the proposed change is that if someone where to add a new context in which backticks could be used for escaping characters, they can't **just** call `Backtick()`.  They have to also know to check the resulting char and `if == '\u'` call the `ScanUnicodeEscapeSequence()` method.  And they might not know because every other escape sequence would work by just calling `Backtick()`.  Not that I'm opposed to this change but it's a potential downside.  Thoughts?